src/sonic-yang-models/yang-models/sonic-local-users-passwords-reset.yang

```diff
@@ -0,0 +1,27 @@
+module sonic-local-users-passwords-reset {
+    yang-version 1.1;
+    namespace "http://github.com/sonic-net/local-users-passwords-reset";
+    prefix local_users_passwords_reset;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "LOCAL_USERS_PASSWORDS_RESET YANG Module for SONiC-based OS";
+    revision 2024-01-04 {
+        description "First Revision";
+    }
+
+    container sonic-local-users-passwords-reset {
```

Comment:
There is a "FEATURE" table for every feature config, maybe consider using that table?

Reply:
why don't we need this table as well, this is the template right?